fabfile/db/__init__.py

```diff
@@ -128,6 +139,15 @@ def run(self, action, suffix=None):
         suffix = re.sub(r'!', r'\\!', suffix)
         suffix = re.sub(r'\$', r'U+0024', suffix)
         suffix = re.sub(r'`', r'\\`', suffix)
+        self.db_cfg['NAME'] = self.name or self.db_cfg['NAME']
```

Comment:
It is not efficient. Rewrite as:
  
    if self.name:
        self.db_cfg['NAME'] = self.name

In this case python call three unnecessary instructions:

    LOAD_FAST             0 (db_cfg)
    LOAD_CONST            2 ('NAME')
    BINARY_SUBSCR 

`BINARY_SUBSCR` is optimized only for Python lists. For dict slow `PyObject_GetItem` is called.

Same below.